bsockMiddleware.js

```diff
@@ -35,7 +35,8 @@ export default function bsockMiddleware (options) {
           dispatch({ type: 'SOCKET_ERROR', payload: err });
         });
 
-        socket.on('connect', () => {
+        socket.on('connect', async () => {
+          await socket.call('auth', apiKey);
```

Comment:
Hopefully the socket was initialized with `ssl`

Reply:
😬